parm/soca/obs/config/adt_3a_egm2008.yaml

```diff
@@ -3,11 +3,11 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: ${OBS_DIR}/${OBS_PREFIX}adt_3a_egm2008.${OBS_DATE}.nc4
+      obsfile: !ENV ${DATA}/obs/${OPREFIX}adt_3a_egm2008.${CDATE}.nc4
```

Comment:
can we please use `${PDY}${cyc}` instead of `${CDATE}`?
We will eventually deprecate the use of `${CDATE}` in favor of the above.

Reply:
@CoryMartin-NOAA @guillaumevernieres I presume this would require changes in the subject of the previous PR?

Reply:
Just an FYI.
There has been a long standing issue https://github.com/NOAA-EMC/global-workflow/issues/198 about this.

Reply:
OR, since these aren't tested, do the PDYcyc change here as a 'gentle reminder' and we do the sed command on the rest soon.

Reply:
One is almost as easy as the other.

Reply:
They aren't tested (yet ...) @CoryMartin-NOAA but we would like to use them in the stand alone application ... Which probably need to be revived too. I say we keep it the way it is right now and deal with `cdate` issue an other day.

Reply:
> can we please use `${PDY}${cyc}` instead of `${CDATE}`? We will eventually deprecate the use of `${CDATE}` in favor of the above.

see [GDASApp #237](https://github.com/NOAA-EMC/GDASApp/issues/237) , good enough for now @aerorahul ?

Reply:
@aerorahul @guillaumevernieres It's now with the ${PDY}${cyc} format.